web/package.json

```diff
@@ -2,6 +2,25 @@
   "name": "web",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
```

Comment:
The test command is missing

Reply:
After 3 hours of trying to make vitest, react testing library and react router work together, I give up. I'm gonna need some help on this one.